src/styles/_global-styles.scss

```diff
@@ -1,3 +1,7 @@
+// :root {
+//     --green1: #4de0b4;
+// }
```

Comment:
Can we remove this commented out code?  It doesn't appear to have a purpose any longer.